client/components/ItemActionConfirmation/forms/updateAssignmentForm.jsx

```diff
@@ -72,7 +72,7 @@ class Component extends React.Component {
             {
                 key: 'Slugline:',
                 value: (
-                    <span className="ItemActionConfirmation__metadata__slugline">
+                    <span className="MetadataView__metadata__slugline">
```

Comment:
We can keep `Metadataview` css class. But do we need other classes like `MetadataView__metadata__slugline`?

Reply:
I had kept it as a part to do when we refactor updateAssignments. But have changed it now to use <preview/row> component instead and taken out MetadataView__metadata__slugline.